artifactory/commands/transferfiles/transfer.go

```diff
@@ -198,6 +198,13 @@ func (tdc *TransferFilesCommand) transferSingleRepo(sourceRepoKey string, target
 			return
 		}
 	}
+	// Performing a reset on the source if chunks were not removed
```

Comment:
// Performing a reset on the source if chunks were not removed
-->
// Ensure the data structure which stores the upload tasks on Artifactory's side is wiped clean, in case some of the requests to delete handles tasks sent by JFrog CLI did not reach Artifactory.

Reply:
Done